### ISSUE TITLE
CIFAR10

### ISSUE THREAD

**Issue by hypnopump** [titu1994/Keras-NASNet#4]
Have you managed to train NASNet Cifar on the cifar10 dataset? If yes, which results have you obtained? Thanks in advance.

**Comment by titu1994** [comment]
Sadly, NASNet models are far too large to train on my 4gb GPU. Weights for CIFAR won't be possible. I will however submit a training script which should be very close to the original. 

The only differences would be the cutout augmentation and drop path regularization - which may affect performance.

**Comment by titu1994** [comment]
Weights for ImageNet are now available. The size of the input must be 224 or 331 however.

**Comment by hypnopump** [comment]
I've tried it on Crestle (2 hours of free GPU Nvidia K80 per account and no credit card required) - https://crestle.com and it doesn't seem to work properly as described in the paper... I'll try another time and see

**Comment by titu1994** [comment]
This may be due to their use of stochastic droppath as well as dropout for all Cifar models. 

That + the cutout augmentation seems to be rather important to get a noticeable increase in performance.
